$NetBSD: patch-scripts_Makefile.am,v 1.4 2023/01/31 07:59:28 khorben Exp $

Fix the installation path for configuration scripts and vm-support

--- scripts/Makefile.am.orig	2021-08-31 09:02:49.000000000 +0000
+++ scripts/Makefile.am
@@ -20,12 +20,13 @@ defaultscripts += poweroff-vm-default
 defaultscripts += suspend-vm-default
 defaultscripts += resume-vm-default
 
-confdir = /etc/vmware-tools
+EGDIR = @datarootdir@/examples
+confdir = ${EGDIR}/vmware-tools
 
 conf_SCRIPTS = ./common/statechange.subr
 conf_SCRIPTS += $(defaultscripts)
 
-vmsupportdir = /usr/bin
+vmsupportdir = ${PREFIX}/bin
 
 vmsupport_SCRIPTS = ./common/vm-support
 
